docker/README.md

```diff
@@ -73,21 +75,21 @@ distribution using debians.
    image of Gazebo Garden:
 
     ```
-    ./build.bash gz-garden ./Dockerfile.gz
+    ./build.bash gz ./Dockerfile.gz
```

Comment:
The instruction above says the first argument must be the name of the Gazebo distribution. Should this change be reverted?

Reply:
Guess I used auto-replacement

I'll check everything and send update